src/main/java/org/opentripplanner/transit/model/network/CarAccess.java

```diff
@@ -0,0 +1,11 @@
+package org.opentripplanner.transit.model.network;
+
+/**
+ * GTFS codes:
+ * 0 = unknown / unspecified, 1 = cars allowed, 2 = cars NOT allowed
```

Comment:
Since this is a class in the main model, it would be better to document what the class is used for than what the gtfs mapping is.

I think it would make sense to say that this describes whether cars are allowed on-board a trip.

Reply:
I created this comment:
```
// This represents the state of whether cars are allowed on board trips.
// If the state is unknown, cars are not allowed.
```

I also added this comment to BikeAccess, which only had similar information to the GTFS codes previously:
```
// This represents the state of whether bikes are allowed on board trips (or routes).
// If the state is unknown, bikes are not allowed.
```

Reply:
What do you think about removing the second line about that unknown means not allowed? It feels like it is up to how other parts of the code wants to interpret the unknown value.

Reply:
I removed the second line